UnityProject/Assets/Scripts/UI/Core/Net/Page/NetPageSwitcher.cs

```diff
@@ -66,7 +66,7 @@ public override void Init()
 				}
 			}
 
-			if (MasterTab.IsServer && !StartInitialized)
+			if (containedInTab.IsMasterTab && !StartInitialized)
```

Comment:
don't use ! unless you're inversing something
```cs
StartInitialized == false
```

Reply:
to be fair is old code